s3-cloudfront-spa/example-pattern.json

```diff
@@ -0,0 +1,74 @@
+
+{
+  "title": "AWS CloudFront and S3 for Single Page Application Hosting",
+  "description": "This sample project demonstrates how to use AWS CloudFront and S3 to host a single-page application (SPA). The pattern creates a CloudFront distribution that serves the SPA content from an S3 bucket configured for static website hosting.",
+  "level": "200",
+  "framework": "terraform",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
```

Comment:
Please reword this to be no more than 4-6 full sentences

Reply:
Updated the json and reduced to 6 steps